### PR TITLE
ES translations

### PR DESCRIPTION
**Pull request by criadoperez**
Added translations so ES can be launched for version 3.4. (reached the 20 left threshold)
If this pull request is accepted I'll add the rest in a few days before the deadline of the 15th of December.

### CODE REVIEW

**Comment by TurboGit** [comment]
Thanks !

File: po/es.po [hunk 1]
```diff
@@ -1904,6 +1904,7 @@ msgstr "FALSO"
 #: ../build/bin/preferences_gen.h:2945
 msgid "hides built-in presets of utility modules in presets menu."
 msgstr ""
+"ocultar los preajestes preinstalados para los módulos de funcionalidad."
```
Comment (rgo) [inline po/es.po:1907]: Typo error: preajestes instead of preajustes

File: po/es.po [hunk 2]
```diff
@@ -2604,7 +2614,7 @@ msgstr ""
 
 #: ../build/bin/preferences_gen.h:4785
 msgid "auto-apply chromatic adaptation defaults"
-msgstr ""
+msgstr "auto-aplicar aberración cromática por defecto"
```
Comment (rgo) [inline po/es.po:2617]: I'd say it doesn't mean the same 🤔

File: po/es.po [hunk 3]
```diff
@@ -5331,7 +5355,7 @@ msgstr " (registro)"
 
 #: ../src/develop/blend_gui.c:1539
 msgid "reset to default blend colorspace"
-msgstr ""
+msgstr "resetear la mezcla de espacio de color por defecto"
```
Comment (rgo) [inline po/es.po:5358]: I'd say "... a la mezcla... " because you're not resetting the blend itself

File: po/es.po [hunk 4]
```diff
@@ -5576,7 +5600,7 @@ msgstr "factor de intensidad"
 
 #: ../src/develop/blend_gui.c:2068
 msgid "adjust the boost factor of the channel mask"
-msgstr ""
+msgstr "adjustar el factor incremental de la máscara de canal"
```
Comment (rgo) [inline po/es.po:5603]: Typo: fix the verb (ajustar)

File: po/es.po [hunk 5]
```diff
@@ -12980,6 +13025,8 @@ msgstr "gestor de máscaras"
 #: ../src/iop/monochrome.c:97
 msgid "quickly convert an image to black & white using a variable color filter"
 msgstr ""
+"convertir rápidamente una imagen a blancoy negro usando un filtro de color "
```
Comment (rgo) [inline po/es.po:13028]: Missing space between "blanco y negro"

File: po/es.po [hunk 6]
```diff
@@ -14192,6 +14256,8 @@ msgstr ""
 #: ../src/iop/toneequal.c:319
 msgid "relight the scene as if the lighting was done directly on the scene"
 msgstr ""
+"reiluminar la escena como si la ilumnicación se hiciera directamente en la "
```
Comment (rgo) [inline po/es.po:14259]: Typo: iluminación

**Comment by rgo** [comment]
@criadoperez thanks for your time. I've left some comments in your translation (some typos). I couldn't review everything, sorry.

**Pull request merged**